frontend_app_admin_workspace_user/webpack.tracim.config.js

```diff
@@ -0,0 +1,2 @@
+module.exports = require('./webpack.config.js')
+module.exports.externals = { tracim_frontend_lib: 'tracim_frontend_lib', ...require('tracim_frontend_vendors/dist/externals.json') }
```

Comment:
can you explain why you are moving this part here ?

Reply:
You asked that we use separate webpack configs for the two build flavors. Do you still agree with this?

Reply:
I do, but I'm not sure about all the implications

Reply:
Okay, we'll talk about it :-)